app/javascript/components/rooms/RoomView.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios"
+import {Row, Col, Tabs, Tab} from "react-bootstrap";
+import ButtonLink from "../stylings/buttons/ButtonLink";
+import FeatureTabs from "./FeatureTabs";
+import {Link, useParams} from "react-router-dom";
+import {useQuery} from "react-query";
+import { Spinner } from "../stylings/Spinner"
+import {House} from "react-bootstrap-icons";
+
+export default function RoomView() {
+  const { friendly_id } = useParams()
+
+  const { isLoading, error, data: room, isFetching } = useQuery("getRoom", () =>
```

Comment:
@hadicheaito1 What do you think of this?

Reply:
Yeah that looks clean :+1: